Wallet/Cryptography/SqlConnector.cs

```diff
@@ -74,7 +90,7 @@ public SqlConnector(string userId, string password, string initialCatalog, strin
                 try
                 {
                     connection.Open();
-                    using (SqlCommand command = new SqlCommand(string.Format(GetPrivateKeyByIdQueryTempalte, identifier), connection))
+                    using (SqlCommand command = new SqlCommand(string.Format(GetPrivateKeyByIdQueryTemplate, identifier), connection))
```

Comment:
I would've add a method to verify that 'identifier' doesn't contains special chars (sql injection...).
i guess its safe to assume that identifier won't have " **'** " ....

Reply:
good point! tnx
fix it